index.ts

```diff
@@ -18,14 +18,14 @@ if (argv._.length === 0 || argv['h'] || argv['help']) {
   handleDefaultDownload(argv._[0], argv['f'])
 }
 
-function handleDev(gitTagOrBranch?: string) {
-  const url = gitTagOrBranch
-    ? `https://raw.githubusercontent.com/microsoft/vscode/${gitTagOrBranch}/src/vs/vscode.proposed.d.ts`
```

Comment:
Unfortunately moving these files will cause users of previous versions of vscode-dts to break. Though ultimately they'll have to at some point as we cannot support the old URL forever.

When we update vscode-dts we should include an [automatic update check](https://www.npmjs.com/package/update-check) to warn in case things change again in the future.